<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWizzardDialog</class>
 <widget class="QDialog" name="MapWizzardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Wizzard</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="m_titleLabel">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="text">
            <string>Title :</string>
           </property>
           <property name="buddy">
            <cstring>m_titleLineedit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_titleLineedit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="m_colorLabel">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="text">
            <string>Background Color :</string>
           </property>
           <property name="buddy">
            <cstring>m_colorButton</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_colorButton">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Permission Mode:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_permissionComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Shape</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QRadioButton" name="m_landscapeButton">
              <property name="text">
               <string>Landscape</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_portraitButton">
              <property name="text">
               <string>Portrait</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_squareButton">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Square</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Grid</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Grid Pattern:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Color:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Size:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Scale:</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QComboBox" name="m_gridPattern">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="ColorButton" name="m_gridColorBtn">
                <property name="text">
                 <string>Color</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="m_sizeGrid">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>200</number>
                </property>
                <property name="value">
                 <number>50</number>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QSpinBox" name="m_scaleOfGrid">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>999</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="m_unitPattern">
                  <item>
                   <property name="text">
                    <string>m</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>cm</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>inch</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>feet</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>px</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Dimensions (in pixels)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="m_smallSizeButton">
          <property name="text">
           <string>Small</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_mediumSizeButton">
          <property name="text">
           <string>Medium</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_bigSizeButton">
          <property name="text">
           <string>Big</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_hugeSizeButton">
          <property name="text">
           <string>Huge</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_customSize">
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="m_widthLabel">
            <property name="text">
             <string>Width :</string>
            </property>
            <property name="buddy">
             <cstring>m_widthEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_widthEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="inputMask">
             <string>00009</string>
            </property>
            <property name="maxLength">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="m_heightLabel">
            <property name="text">
             <string>Height :</string>
            </property>
            <property name="buddy">
             <cstring>m_heightEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_heightEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="inputMask">
             <string>00009</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>228</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>widgets/colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MapWizzardDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MapWizzardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_widthEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_heightEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
